Packs/Phishing/Playbooks/Process_Email_-_Generic_v2.yml

```diff
@@ -564,8 +564,17 @@ tasks:
           accessor: Keywords
       emailmessageid:
         complex:
-          root: Email
-          accessor: HeadersMap.Message-ID
+          root: incident.emailheaders
+          filters:
+          - - operator: isEqualString
+              left:
+                value:
+                  simple: incident.emailheaders.headername
+                iscontext: true
+              right:
+                value:
+                  simple: Message-ID
```

Comment:
Maybe it's a good idea to ignore-case as well, to future proof our playbook in case new integrations will not keep the capitalization standard.

Reply:
In this case we don't want to ignore case since the headers will always be in the same format.